src/man/include/failover.xml

```diff
@@ -71,6 +71,20 @@
             </citerefentry>,
             manual page.
             <variablelist>
+                <varlistentry>
+                    <term>
+                        dns_resolver_server_timeout
```

Comment:
I'm sorry I didn't notice this before, but you need to add the new option to `src/config/cfg_rules.in `othwrwise anyone trying to use it will get a warning. You'll also want to add the option to ``src/config/SSSDConfig/__init__.py.in``, `src/config/SSSDConfigTest.py` and `src/config/etc/sssd.api.conf`

Reply:
Added.